### PR TITLE
Update labels.json

### LINKED ISSUE #20: Complete the labels
**Description**

We don't have all the labels [mentioned](https://opensource.creativecommons.org/contributing-code/repo-labels/)

**Priority**: Medium

**Screenshots**

**Additional Context**    

Join Eddie's discord community [here](http://discord.eddiehub.org)

### PR DESCRIPTION
**Pull request by Sonichigo**
Resolve #20 
We now have all the labels mentioned in [Repo-labels](https://opensource.creativecommons.org/contributing-code/repo-labels/)

## Proposed Changes
🏷 status: label work required
🙅 status: discontinued

### CODE REVIEW

File: labels.json [hunk 1]
```diff
@@ -87,6 +87,14 @@
     "name": "🧹 status: ticket work required",
     "color": "666666"
   },
+  {
+    "name": "🙅 status: discontinued",
+    "color": "666666"
```
Comment (Panquesito7) [inline labels.json:92]: Why use the same color here and in line 96? I would suggest using this color `#0033CC`.
```suggestion
    "color": "#0033CC"
```

File: labels.json [hunk 2]
```diff
@@ -87,6 +87,14 @@
...
+  },
+  {
+    "name": "🏷 status: label work required",
+    "color": "666666"
```
Comment (Panquesito7) [inline labels.json:96]: Here I suggest using this color `#F4EE00`.
```suggestion
    "color": "#F4EE00"
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: labels.json (hunk 1 above)
Comment (Sonichigo) [reply labels.json:92]: Cool, i followed the previous repo-labels and used the same colors! This blue looks better!

File: labels.json (hunk 2 above)
Comment (Sonichigo) [reply labels.json:96]: Sure!

**Review by Panquesito7 — APPROVED** [review]
LGTM. Thank you for your contribution! 😄👍

**Review by krishdevdb — CHANGES_REQUESTED** [review]
Maybe for the colors we shall adhere to the guidelines since we are stating we use them

**Review by krishdevdb — COMMENTED** [review]
ik, this is a out of box change but really needed

We use the label `no-issue-activity` and `no-pr-activity` on all the repos, but we have only `no issue activity` that is also without `-` so maybe consider fixing this

**Comment by Sonichigo** [comment]
Were using the color stated in guidelines previously!! Will update it!

File: labels.json [hunk 3]
```diff
@@ -87,6 +87,14 @@
     "name": "🧹 status: ticket work required",
     "color": "666666"
   },
+  {
+    "name": "🙅 status: discontinued",
```
Comment (eddiejaoude) [inline labels.json:91]: Is this the same as `discarded` https://github.com/EddieHubCommunity/gh-action-open-source-labels/blob/df3b134d2c81e45ba6938abb0e64b1c311907d20/labels.json#L15

Comment (Sonichigo) [reply labels.json:91]: No No, it's like discontinued is more like it's on halt for now and will be picked by after high priority ones!!

Comment (nhcarrigan) [reply labels.json:91]: The discontinued label is intended for any outstanding issues when a repository is archived or otherwise no longer maintained.

**Review by Panquesito7 — APPROVED** [review]

**Review by FayasNoushad — APPROVED** [review]

**Pull request merged**